libraries/chain/database.cpp

```diff
@@ -296,6 +296,36 @@ bool database::is_known_transaction( const transaction_id_type& id )const
    return trx_idx.find( id ) != trx_idx.end();
 } FC_CAPTURE_AND_RETHROW() }
 
+//uint32_t database::find_transaction( const transaction_id_type& id, const time_point_sec& expiration_time ) const
+uint32_t database::find_transaction( const transaction_id_type& id, const time_point_sec& expiration_time )const
```

Comment:
There is nothing here that requires this be done in database. It can just as easily be done in `database_api`

Reply:
Moved to database_api